### PR TITLE
Research completed for: shop/travel_agency|Thomson

### LINKED ISSUE #1833: Research needed for: shop/travel_agency|Thomson
#### :thinking: We need help deciding what to do with "shop/travel_agency|Thomson"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/travel_agency|Thomson` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/travel_agency|Thomson`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/travel_agency|Thomson`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### PR DESCRIPTION
**Pull request by AlbyIanna**
closes #1833 
I also corrected the name from _Thomson_ to _Thom**p**son_

### CODE REVIEW

File: config/canonical.json [hunk 1]
```diff
@@ -32613,11 +32613,13 @@
       "shop": "travel_agency"
     }
   },
-  "shop/travel_agency|Thomson": {
+  "shop/travel_agency|Thompson": {
```
Comment (ENT8R) [inline config/canonical.json]: Correcting the name here does not use that much. If somebody builds the files again using `npm run build` the same name as before is added again.

Comment (AlbyIanna) [reply config/canonical.json]: Oh, I didn't notice, sorry!

**Comment by AlbyIanna** [comment]
@ENT8R, I changed back the entry to `Thomson`

File: config/canonical.json [hunk 2]
```diff
@@ -32616,8 +32616,10 @@
   "shop/travel_agency|Thomson": {
     "count": 113,
     "tags": {
-      "brand": "Thomson",
-      "name": "Thomson",
+      "brand": "The Thompson Travel Agency",
+      "brand:wikidata": "Q1170162",
+      "brand:wikipedia": "en:The Thompson Travel Agency",
+      "name": "The Thompson Travel Agency",
```
Comment (ENT8R) [inline config/canonical.json]: Did you really read the [wikidata entry](https://www.wikidata.org/wiki/Q1170162)? This links to a book written by Jules Verne but not to a travel agancy called 'Thomson'... 😄
So please change back the `name` and the `brand` tag, and maybe change the `brand:wikipedia` tag to https://en.wikipedia.org/wiki/TUI_UK (if that is the correct entry...) and the `brand:wikidata` tag to [Q7795876](https://www.wikidata.org/wiki/Q7795876) (if that is the correct entry...)

Comment (AlbyIanna) [reply config/canonical.json]: what exactly should I put into `brand:wikipedia`? Not the link right?

Comment (AlbyIanna) [reply config/canonical.json]: i committed a change, but i'm not sure it's correct

File: config/canonical.json [hunk 3]
```diff
@@ -32616,10 +32616,10 @@
   "shop/travel_agency|Thomson": {
     "count": 113,
     "tags": {
-      "brand": "The Thompson Travel Agency",
-      "brand:wikidata": "Q1170162",
-      "brand:wikipedia": "en:The Thompson Travel Agency",
-      "name": "The Thompson Travel Agency",
+      "brand": "Thomson",
+      "brand:wikidata": "Q7795876",
+      "brand:wikipedia": "en:TUI_UK",
```
Comment (bhousel) [inline config/canonical.json]: Needs to be `"en:TUI UK"`  (no underscore `_`)
Otherwise looks good 👍

**Comment by ENT8R** [comment]
I corrected it by myself right now 😄
Thank you for this PR @AlbyIanna 🎉

**Pull request merged**